src/utils.js

```diff
@@ -35,7 +35,17 @@ function makeDuckDuckGo(query) {
 }
 
 function handleURL(rawURL) {
-  if (rawURL.startsWith(IPFS_PREFIX) || rawURL.startsWith(IPNS_PREFIX) || rawURL.startsWith(HYPER_PREFIX) || rawURL.startsWith(WEB3_PREFIX)) {
+  if (rawURL.endsWith('.eth')) {
+    if (!rawURL.startsWith(IPFS_PREFIX)) {
+      return `${IPFS_PREFIX}${rawURL}`;
```

Comment:
Should this be IPNS? Also this would mean that HTTPS `.eth` domains wouldn't work right?

Reply:
FYI, HTTPS `.ens`, where the cert is verified by the blockchain itself, would be ideal.

That's what actually makes ENS worth having.